examples/bzlmod/BUILD.bazel

```diff
@@ -43,3 +44,11 @@ py_test_with_transition(
     main = "test.py",
     deps = [":lib"],
 )
+
+alias(
+    name = "yamllint",
+    # This is using the struct defined by the spoke repo 'yamllint' and it is
+    # re-exported by the hub repo named 'pip'. This allows bzlmod and
+    # non-bzlmod users to access the entry point targets.
+    actual = yamllint_bin.yamllint,
```

Comment:
I don't really follow this part -- why does it have to be put into a bzl file and loaded like this? This is pointing to a target, right? So why can't it just be "@pip//yamllint:bin"?

(I'm guessing the answer has something to do with bzlmod, repo mapping, and repo visibility, but I'd like to understand more).